docs/jazzy.yaml

```diff
@@ -6,30 +6,17 @@ exclude:
   - "**/*.h"
   - "**/*-ObjC.swift"
   - "**/*Deprecations.swift"
-  - "**/*SwiftExtensions.swift"
-
-copyright: 'Copyright (c) 2018-present, salesforce.com, inc. All rights reserved. Redistribution and
-use of this software in source and binary forms, with or without modification, are permitted provided
-that the following conditions are met:
- * Redistributions of source code must retain the above copyright notice, this list of conditions
- and the following disclaimer.
- * Redistributions in binary form must reproduce the above copyright notice, this list of
- conditions and the following disclaimer in the documentation and/or other materials provided
- with the distribution.
- * Neither the name of salesforce.com, inc. nor the names of its contributors may be used to
- endorse or promote products derived from this software without specific prior written
- permission of salesforce.com, inc.
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
-IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
-FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
-CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
-WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
-WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.'
+  - "**/SwiftExtensions.swift"
+readme: 'libs/SalesforceSwiftSDK/readme.md'
+copyright: '© Copyright 2017–Current salesforce.com, inc. All rights reserved. Salesforce is a registered trademark of salesforce.com, inc.,
```

Comment:
@trooper2013 Maybe we should just remove the `copyright` section here. It's just doc. I know I mentioned adding it in last time around ;-)

Reply:
@bhariharan This copyright shows up in the footer of documentation pages. I think we should keep it.

Reply:
Sounds good.